### ISSUE TITLE
Clarify instructions?

### ISSUE THREAD

**Issue by ericlaw1979** [jclehner/nmrpflash#45]
Thanks for a wonderful tool!

If it matters, when you say to connect the PC to the router via ethernet, mention it should go in the blue (downstream) ports, not the yellow (WAN) port?

I wasn't able to get the tool to work:

C:\Users\ericlaw\Downloads\routerfix>nmrpflash -i net0 -f R6700v3-V1.0.4.98_10.0.71.chk
Failed to bind after 5 seconds: bind: The requested address is not valid in its context.

C:\Users\ericlaw\Downloads\routerfix>nmrpflash -i net1 -f R6700v3-V1.0.4.98_10.0.71.chk
Failed to bind after 5 seconds: bind: The requested address is not valid in its context.

... until I manually edited my IPv4 settings to use a fixed IP address (as documented in https://kb.netgear.com/000059633/How-to-upload-firmware-to-a-NETGEAR-router-using-TFTP-client)

**Comment by jclehner** [comment]
Hi, I've updated the instructions a bit. The `Failed to bind after 5 seconds.` message isn't a bug per se, but the next release should fix this issue.

**Comment by snibbo71** [comment]
Sorry this is a bit old, but I found the only way to get around this error was to plug into another ethernet hub/switch and then into the router port for flashing a R6260 on Windows 10 on a Surface Pro 3 docking station. Otherwise, since the port wasn't powered it kept throwing the above error even with a manually configured address.

But once I connected via a separate switch the whole thing worked and seriously saved my bacon - thank you!
